pyproject.toml

```diff
@@ -7,3 +8,42 @@ requires = [
     "pysam",
 ]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "cgat"
+version = "0.7.5"
```

Comment:
This might have needed changing to

```
dynamic = ["version"]
```

...to get `setuptools_scm` working.

Sorry for not adding that yesterday, can wait and be addressed as a separate task though, see #137